tcms/testruns/tests/test_admin.py

```diff
@@ -91,3 +96,48 @@ def test_changelist_view_icon_column(self):
             """
             ),
         )
+
+    @parameterized.expand(
+        [
+            ("positive_tests", positive),
+            ("neural_tests", neutral),
+            ("negative_tests", negative),
```

Comment:
It is clear that this is a test so the suffix "_tests" doesn't bring anything new nor useful. Remove it.